integration_tests/features/environment.py

```diff
@@ -13,4 +14,5 @@ def after_all(context):
 
 
 def after_scenario(context, scenario):
-    os.system("rm -rf mock/temp/*")
+    if not hasattr(context, "temp_dir"):
```

Comment:
should this be `if hasattr`?

Reply:
so this is an indicator if the tests run with the new style or the old style, we dont want to do this if it ran with the new style..